vpr/src/draw/draw_triangle.h

```diff
@@ -0,0 +1,49 @@
+#ifndef DRAW_TRIANGLE_H
+#define DRAW_TRIANGLE_H
+
+#include <cstdio>
+#include <cfloat>
+#include <cstring>
+#include <cmath>
+#include <algorithm>
+#include <sstream>
+#include <array>
+#include <iostream>
+
+#include "vtr_assert.h"
+#include "vtr_ndoffsetmatrix.h"
+#include "vtr_memory.h"
+#include "vtr_log.h"
+#include "vtr_color_map.h"
+#include "vtr_path.h"
+
+#include "vpr_utils.h"
+#include "vpr_error.h"
+
+#include "globals.h"
+
+#include "move_utils.h"
+
+#ifndef NO_GRAPHICS
+
+#    include "draw_global.h"
+
+#    include "ezgl/point.hpp"
+#    include "ezgl/application.hpp"
+#    include "ezgl/graphics.hpp"
+#    include "draw_color.h"
+#    include "search_bar.h"
+#    include "draw_debug.h"
+#    include "manual_moves.h"
+
+#    include "rr_graph.h"
+#    include "route_util.h"
+#    include "place_macro.h"
+#    include "buttons.h"
+
+void draw_triangle_along_line(ezgl::renderer* g, ezgl::point2d start, ezgl::point2d end, float relative_position = 1., float arrow_size = DEFAULT_ARROW_SIZE);
```

Comment:
Move function-level comments from .cpp file to here (the how to call and what do I do part).